specification/ai/Azure.AI.Projects/evaluation-results/models.tsp

```diff
@@ -0,0 +1,74 @@
+import "@typespec/rest";
+import "@azure-tools/typespec-autorest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+import "../common/models.tsp";
+import "../main.tsp";
+
+using TypeSpec.OpenAPI;
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+using Azure.Core.Traits;
+
+namespace Azure.AI.Projects;
+
+@doc("Evaluation Result resource Definition")
+@discriminator("ResultType")
```

Comment:
why do we need the discriminator? We only need that if `EvaluationResult` is polymorphic and has multiple sub-types